nsdb-core/src/main/scala/io/radicalbit/nsdb/commit_log/StandardCommitLogSerializer.scala

```diff
@@ -95,17 +95,17 @@ class StandardCommitLogSerializer extends CommitLogSerializer with TypeSupport {
     * @param clazz value class canonicalName
     * @return deserialized value instance casted into the correct class
     */
-  private def argument(clazz: String): AnyRef = {
-    val longClazz: String = classOf[java.lang.Long].getCanonicalName
-    val intClazz          = classOf[java.lang.Integer].getCanonicalName
-    val doubleClazz       = classOf[java.lang.Double].getCanonicalName
-    val stringClazz       = classOf[java.lang.String].getCanonicalName
+  private def argument(clazz: String): NSDbType = {
+    val longClazz: String = classOf[NSDbLongType].getCanonicalName
```

Comment:
why just this with type annotation?

Reply:
I guess it's the result of a bad copy and paste. There isn't a valid reason for that.